app/models/foreman_docker/service/actions/container/start.rb

```diff
@@ -0,0 +1,30 @@
+module ForemanDocker
+  module Service
+    module Actions
+      module Container
+        class Start < ::Actions::EntryAction
+          def plan(container)
+            action_subject(container)
+            plan_self(:container => container.id)
+          end
+
+          def run
+            container = ::Container.find(input[:container])
+            started = container.compute_resource.create_container(container.parametrize)
+            container.update_attribute(:uuid, started.id) if started
+            started
+          end
+
+          def humanized_name
+            _('Start')
+          end
+
+          def finalize
+            container = ::Container.find(input[:container])
+            Rails.logger.info("[Docker] container #{container.name} successfully started")
```

Comment:
None, just logging. The problem is that I couldn't see in the logs exactly what container was started at a glance, only the id of the task. It's true that you can read that in the dynflow console but I think it's worth adding it to the regular log just in case, it's more straightforward for users.